Dockerfile.desktop

```diff
@@ -55,16 +61,23 @@ COPY . /code
 
 WORKDIR /code
 
-# Install Firefox and Geckodriver
-RUN INSTALLER_DOWNLOAD_URL=https://raw.githubusercontent.com/hackebrot/install-firefox/master/install-firefox.sh \
-    && curl $INSTALLER_DOWNLOAD_URL -sSf | sh -s -- --firefox $FIREFOX_VERSION --geckodriver $GECKODRIVER_VERSION
-
-# Update to firefox nightly
-ARG FIREFOX_DOWNLOAD_URL=https://download.mozilla.org/?product=firefox-nightly-latest-ssl&lang=en-US&os=linux64
+# Download Firefox
+ARG FIREFOX_DOWNLOAD_URL=https://download.mozilla.org/?product=firefox${FIREFOX_VERSION}-latest-ssl&lang=en-US&os=linux64
 RUN wget --no-verbose -O /tmp/firefox.tar.bz2 $FIREFOX_DOWNLOAD_URL \
-    && mkdir utilities/firefox-nightly \
-    && tar -C utilities/firefox-nightly -xjf /tmp/firefox.tar.bz2 \
-    && rm /tmp/firefox.tar.bz2
+    && rm -rf /opt/firefox \
+    && tar -C /opt -xjf /tmp/firefox.tar.bz2 \
+    && rm /tmp/firefox.tar.bz2 \
+    && ln -fs /opt/firefox/firefox /usr/bin/firefox
+
+# @hackebrot
+ARG GECKODRIVER_DOWNLOAD_URL=https://github.com/mozilla/geckodriver/releases/download/v$GECKODRIVER_VERSION/geckodriver-v$GECKODRIVER_VERSION-linux64.tar.gz
+RUN rm -rf /opt/geckodriver \
+    && curl -sSL $GECKODRIVER_DOWNLOAD_URL -o /tmp/geckodriver.tar.gz \
+    && tar -C /opt -zxf /tmp/geckodriver.tar.gz \
+    && rm /tmp/geckodriver.tar.gz \
+    && mv /opt/geckodriver /opt/geckodriver-$GECKODRIVER_VERSION \
```

Comment:
Why rename the folder/binary?
Are we installing multiple different versions and we're trying to avoid collisions? Although we do symlink it below on L80, which removes the versioning anyways.

Reply:
I don't quite remember why I did this. I think I am replacing the installed geckodriver which can often be out of date.